src/components/ui/FiltersPart/sections/PriceRangeSection.tsx

```
@@ -0,0 +1,29 @@
+}
+
+export const PriceRangeSection = ({ priceRange, onChange }: Props) => (
+  <div className="space-y-2">
+    <h3 className="text-sm font-semibold text-gray-900">Price Range</h3>
+
+    <div className="flex gap-2">
+      {(["min", "max"] as const).map((field) => (
+        <div key={field} className="flex-1">
+          <label className="mb-1 block text-xs text-gray-600">
+            {field === "min" ? "From" : "To"}
+          </label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)